src/sentry/services/hybrid_cloud/organization/__init__.py

```diff
@@ -177,6 +179,10 @@ class ApiOrganizationFlags:
 
 @dataclass
 class ApiOrganizationSummary:
+    """
+    The subset of organization metadata available from the control silo specifically.
+    """
+
     slug: str = ""
     id: int = -1
     name: str = ""
```

Comment:
@mikejihbe 
Ideally, the control silo can provide all this metadata so that the user serializer does not require a span out.  Specifically, the user silo requires the name of every user organization as part of its payload to the frontend, so we will eventually need to add this into your organization mapping table.  I feel confident this is all the metadata we will need, however.